src/test/java/teammates/test/cases/webapi/GetFeedbackSessionActionTest.java

```diff
@@ -113,5 +113,28 @@ protected void testAccessControl() throws Exception {
 
         verifyAccessibleForInstructorsOfTheSameCourse(submissionParams);
         verifyAccessibleForAdminToMasqueradeAsInstructor(submissionParams);
+
+        ______TS("only students of the same course can access");
+
+        submissionParams = new String[] {
+                Const.ParamsNames.COURSE_ID, fs.getCourseId(),
+                Const.ParamsNames.FEEDBACK_SESSION_NAME, fs.getFeedbackSessionName(),
+                Const.ParamsNames.INTENT, Intent.STUDENT_RESULT.toString(),
+        };
+
+        verifyAccessibleForStudentsOfTheSameCourse(submissionParams);
+        verifyInaccessibleForStudentsOfOtherCourse(submissionParams);
+
+        ______TS("only instructors of the same course can access");
```

Comment:
This test case has the same description as one already implemented. Try and think about what this new test case is doing that the other isn't already and rename accordingly.

Reply:
Thanks. Just changed it.